src/features/pageObjects/utils/pageObjectTemplate.ts

```diff
@@ -18,14 +20,32 @@ export const getClassName = (title: string) => {
   return className;
 };
 
-export const pageObjectTemplate = (locators: Locator[], title: string, library: ElementLibrary) => {
-  const className = title;
+export const vividusTemplate = (locators: Locator[], pageObject: PageObject) => {
+  let pageCode = `variables.${pageObject.name}.url=(${pageObject.pathname})\n`;
+
+  locators.forEach((it) => {
+    const currentLocatorType = _.camelCase(it.locatorType || pageObject.locatorType || LocatorType.xPath);
+    // @ts-ignore
+    pageCode += `variables.${pageObject.name}.${it.type}.${it.name}=By.${currentLocatorType}(${it.locator[currentLocatorType]})\n`;
+  });
+
+  return { pageCode, title: pageObject.name };
+};
+
+export const pageObjectTemplate = (locators: Locator[], pageObject: PageObject) => {
```

Comment:
only 3 fields from the PageObject are used here, I'm not sure that passing the whole object with all the fields is needed here.